app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<section class="min-h-screen bg-gray-200 flex flex-col items-center w-full">
+  <div class="bg-main w-full text-white pt-2">
+    <%= link_to root_path, class: "text-white ps-3" do %>
+      <i class="fa-solid fa-arrow-left fa-lg"></i>
+    <% end %>
+    <div class="h-10 bg-main w-full text-white flex justify-center items-end pb-2">
+      <h1 class="font-bold">Add a new category</h1>
+    </div>
+  </div>
+  <div class="mt-4 w-full px-3 flex flex-col justify-center">
+    <%= form_with(model: @category, url: categories_path) do |form| %>
+      <div class="flex flex-col gap-2 bg-white">
+        <div class="flex items-center">
+          <div class="h-20 w-20 p-2">
+            <%= image_tag(@category.icon, class: "h-full") %>
+          </div>
+          <h1 class="ms-4 font-bold"><%= @category.name %> Transactions</h1>
+        </div>
+        <table>
+          <thead>
+            <tr class="flex px-4 justify-between">
+              <th>Date</th>
+              <th>Amount</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @transactions.each do |transaction| %>
+              <tr class="flex px-4 justify-between">
+                <td><%= transaction.created_at %></td>
+                <td><%= transaction.amount %></td>
+              </tr>
+            <% end %>
+          </tbody>
+        </table>
+        <p class="px-4 font-bold flex justify-end">Total Amount: <%= @total_amount %></p>
```

Comment:
- [ ] Here the total amount is required to be at the top as mentioned in the project requirements. 
![image](https://github.com/SamTush/budget/assets/45460192/594294e3-2255-4cdb-bda2-215e269ac699)

### Your Implementation
![image](https://github.com/SamTush/budget/assets/45460192/557327b7-1370-45cf-bf59-13bc26e3633b)